shopinvader_quotation/services/cart.py

```diff
@@ -22,10 +21,12 @@ def request_quotation(self, **params):
         return res
 
     # Validator
+
     def _validator_request_quotation(self):
         return {}
 
     def _convert_one_sale(self, sale):
         res = super()._convert_one_sale(sale)
+        # TODO: Ask simone. Can't find how this is used or why
         res.update({"available_for_quotation": True})
```

Comment:
@sebastienbeau @simahawk any idea what is it for and if it's still needed?

Reply:
@bguillot ping

Reply:
found it https://github.com/shopinvader/documentation/blob/59ed451f385ec1d3d7c7e124fa86a5745737f62a/data/doc_pages/en/api-reference/cart.md

it's only for frontend.
I would add a comment :wink:

Reply:
Thanks ! ;)